src/controller_examples/rabbitmq_controller/proof/liveness.rs

```diff
@@ -744,17 +763,17 @@ proof fn lemma_from_pending_req_in_flight_at_some_step_to_pending_req_in_flight_
         // next_step != RabbitmqReconcileStep::Init,
         forall |rabbitmq_1, resp_o|
```

Comment:
Could you also document on why we need this forall? You explained to me several times but sometimes I still forget and hope there is some doc here. Also, you can rename `rabbitmq_1` to `any_rabbitmq`